conf/test_nobc_nodx.config

```diff
@@ -18,7 +18,7 @@ params {
 
   // Input data to skip both basecalling and demultiplexing
   input = 'https://raw.githubusercontent.com/nf-core/test-datasets/nanoseq/samplesheet_nobc_nodx.csv'
-  protocol = 'cDNA'
+  protocol = 'directRNA'
```

Comment:
Ok, a bit tricky here, when I check the samplesheet_nobc_nodx.csv, I noticed that the fastq files are not all direct cDNA or direct RNA, I guess it wouldn't matter much, we can just use either cDNA or direct RNA

Reply:
Yes, so this means we will have to change the test-data for the `samplesheet_nobc_nodx.csv` because the pipeline is only able to handle 1 `--protocol` at a time.